--- module.go	2019-08-05 13:21:58.655304166 +0200
+++ module.go	2019-08-05 13:23:28.055125298 +0200
@@ -94,12 +94,12 @@

 int bpf_attach_socket(int sock, int fd)
 {
-	return setsockopt(sock, SOL_SOCKET, SO_ATTACH_BPF, &fd, sizeof(fd));
+	return setsockopt(sock, SOL_SOCKET, SO_ATTACH_FILTER, &fd, sizeof(fd));
 }

 int bpf_detach_socket(int sock, int fd)
 {
-	return setsockopt(sock, SOL_SOCKET, SO_DETACH_BPF, &fd, sizeof(fd));
+	return setsockopt(sock, SOL_SOCKET, SO_DETACH_FILTER, &fd, sizeof(fd));
 }

 int bpf_attach_xdp(const char *dev_name, int progfd, uint32_t flags)
